hopp/simulation/hybrid_simulation.py

```diff
@@ -382,16 +387,21 @@ def calculate_installed_cost(self):
             battery_mwh = self.battery.system_capacity_kwh / 1000
 
         # TODO: add tower and trough to cost_model functionality
-        pv_cost, wind_cost, storage_cost, total_cost = self.cost_model.calculate_total_costs(wind_mw,
-                                                                                             pv_mw,
-                                                                                             battery_mw,
-                                                                                             battery_mwh)
+        # pv_cost, wind_cost, storage_cost, total_cost = self.cost_model.calculate_total_costs(wind_mw,
```

Comment:
I think ideally as we integrate full BOS and indirect costs it would be worth rewriting `calculate_total_costs` to be flexible and incorporate those aspects for a hybrid plant. As it stands now `calculate_total_costs` is only applicable to wind, pv and battery based on an outdated study for greenfields and uses an outdated ATB (2020).

Additionally, the method `calculate_total_costs` produces the same costs for each technology as the methods I wrote when using `Singleowner` as the financial model.

Reply:
Ok, if the results are the same we should be good. Thanks.